Update http_server.adoc to talk about headers more. (#71)
redpanda-data/rp-connect-docs: modules/components/pages/inputs/http_server.adoc

You can access these metadata fields using xref:configuration:interpolation.adoc#bloblang-queries[function interpolation].

=== Headers

> Request headers are available as metadata keyed off the header name with no additional prefix. Header name casing is changed during processing to train case:

Reviewer: ```suggestion
Request headers are available as metadata, using the header name as a key with no additional prefix. Header name casing is changed during processing as follows:
```
